templates/simulations/ibm_704_words_explanation.jinja2

```diff
@@ -0,0 +1,92 @@
+    operation, the address, the tag, and in the case of Type A instructions, a decrement, with each
+    of these things occupying a different set of bits.  </p>
+
+    <p>The address describes which register in the general memory that the operation will target.  </p>
+
+    <p>The operation describes the specific action that the computer will take; for example, <code>CLA 100</code>
+    will take the contents of register 100 and replace the accumulator with that value.  Of course,
+    in the machine, the operation wouldn't be represented by <code>CLA</code>, it would be <code>000101000000</code> in
+    memory, or <code>+0500</code> in octal as it is often written in the manuals.  </p>
+
+    <p>The tag is three bits corresponding to the A, B, and C index registers of the IBM 704.  For
+    example <code>100</code> would refer to the A register, <code>010</code> to the B register, and <code>110</code> to both the A
+    and B register.  It is also possible to leave the tag as <code>000</code>, which is like an imaginary
+    register which always has a value of 0.  The effect of the tag is to decrement the target address by
+    whatever is stored in that index register.  For example if index register B (IRB) contains 10,
+    <code>CLA 100, 2</code> would cause <code>CLA</code> to act on register 90 instead of register 100.  (Remember that 2
+    in decimal is 010 in binary).</p>
+
+    <h3 id="typeainstructions">Type A Instructions</h3>
+
+    <p>There are only five Type A instructions.  For these, the first three bits represent the
+    operation.  Type A instructions are distinguished by the computer by the fact that bits 1 and 2
+    are not both 0 (note that bit 1 is the second bit, as bit 0 is the first bit).  Bits 3 - 17
+    contain the numerical value of the decrement, bits 18 - 20 contain the tag, and the remaining
+    bits 21-35 contain the address.  </p>
+
+    <p><img class="img-fluid" src="{{ static('img/simulations/Type_A_Instruction_Structure.png') }}" alt="" /></p>
+
+    <p>If the value of the index register specified by the tag is greater than the decrement, then the
+    index register is reduced by the decrement.  Otherwise, the computer will go back to the address specified.
+    This allows for looping.  </p>
+
+    <h3 id="typebinstructions">Type B Instructions</h3>
+
+    <p>Most instructions are Type B instructions.  All Type B instructions have bits 1 and 2 equal to 0
+    .  The first 12 bits represent the  operation, while the tag and address occupy the same location as in Type A
+        instructions (18-20 and 21-35 respectively).  Bits 12-17 aren't used. </p>
+
+    <p><img class="img-fluid" src="{{ static('img/simulations/Type_B_Instruction_Structure.png') }}" alt="Diagram illustrating use of
+    bits in IBM 704 Type B instruction" /></p>
+
+    <p>Note that the operation has a sign bit.  For whatever reason, it is conventional to refer to
```

Comment:
I think I would get rid of "For whatever reason" here, we can just say that it is conventional so that this can be an objective write-up rather than a subjective one.